src/main/rx/rx.h

```diff
@@ -206,6 +206,9 @@ void setRssiDbmDirect(int16_t newRssiDbm, rssiSource_e source);
 void rxSetRfMode(uint8_t rfModeValue);
 uint8_t rxGetRfMode(void);
 
+void rxSetUplinkTxPwr(uint16_t uplinkTxPwrValue);
```

Comment:
Other developers will have to grep through the code to figure out how to interpret this. Naming it `rxSetUplinkTxPwrMw()` and renaming everything else in a similar way will make this immediately obvious.

Reply:
Agreed. Makes sense. Now is hard to figure what is an enum item index and what is a power value in mW. Working on it.

Reply:
Done.